### DISCUSSION TITLE
Feasibility of Using Momepy-Derived Morphological Values for Residential Building Classification

### DISCUSSION THREAD

**Discussion by zhulfaliant01** [pysal/momepy#552] [Q&A] [answered]
Hey everyone,

I'm currently exploring the potential of utilizing Momepy to compute the morphological values for every building within my region. My objective is to investigate whether it's viable or advisable to construct a residential building classifier based on these morphological attributes.

I've previously experimented with K-Means Clustering for classifying buildings in my city, which yielded promising results. However, I'm now considering the possibility of employing supervised classification methods for this task.

I'd appreciate any insights or recommendations regarding the feasibility and effectiveness of utilizing Momepy-derived morphological values as features for a residential building classifier.

Thank you in advance for your input!

**Comment by martinfleis** [comment, marked as answer]
Hi,

Yeah, that should work. I've been working with both supervised and unsupervised learning based on morphometrics and it usually works, for both classification and regression problems. Obviously you need to model tasks that have some relationship to urban form but then there's no reason not to use it.

For building classification problem, it depends on the actual classes you are trying to predict, whether they express enough morphological variability but my guess is that you can get a pretty decent model out of that.

**Reply by zhulfaliant01** [reply]
Hey Martin, thanks for sharing your insights!

I'd like to inquire further about my project on identifying residential buildings based on their morphological attributes. Could you suggest which morphological characteristics I should prioritize? Additionally, do you have any recommended literature or resources that delve deeper into this subject?

**Reply by martinfleis** [reply]
That very much depends on the local context and the nature of the classification you're trying to predict. Hard to say. You can check some of my papers, the work of Alessandro Araldi, work of Nikola Milojevic-Dupont or stuff that comes from the team of Filip Bijlecki at NUS.
